lib/rucio/web/rest/flaskapi/v1/replicas.py

```diff
@@ -178,7 +176,7 @@ def generate(vo):
         except RucioException as error:
             return generate_http_error_flask(500, error.__class__.__name__, error.args[0])
         except Exception as error:
-            print(format_exc())
+            logging.log(logging.ERROR, "Internal Error", exc_info=True)
```

Comment:
why not use `logging.exception("Internal Error")` like everywhere?

Reply:
Yeah, no idea why I did it differently here. Will fix it.